native/src/components/LayoutedScrollView.tsx

```diff
@@ -7,18 +7,20 @@ import Layout from './Layout'
 type LayoutedScrollViewProps = {
   children?: React.ReactNode
   refreshControl?: React.ReactElement<RefreshControlProps>
+  nestedScrollEnabled?: boolean
```

Comment:
Why is this change needed?

Reply:
@ztefanie while there was a virtual list error saying that `VirtualizedLists should never be nested inside plain ScrollViews with the same orientation because it can break windowing and other functionality`